docs/content/about-iota/iota-wallet/how-to/restore-account.mdx

```diff
@@ -0,0 +1,48 @@
+---
+description: A guide on how to restore an account using a seed or a .stronghold file.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet]
+---
+# Restore From Seed or Stronghold
+
+If you have lost your mnemonic, you can still recover your account provided you have access to your seed or
+`.stronghold` file.
+
+## Backup a Seed
+
+You can export you seed by clicking on the `...` menu, and then clicking `Backup Seed`.
+
+   ![Backup Seed](/img/about-iota/iota-wallet/how-to/restore-account/backup-seed-falafel.png)
+
+Alternatively, you can also generate a seed backup using by clicking on `Electron > Backup Seed`. 
+
+   ![Backup Seed](/img/about-iota/iota-wallet/how-to/restore-account/backup-seed-menu.png)
+
+Firefly will then prompt you to enter your password.
+
+   ![Enter Password](/img/about-iota/iota-wallet/how-to/restore-account/enter-password.png)
+
+With your seed at hand, you can [import your wallet using the IOTA Web Wallet](../getting-started.mdx#importing-an-existing-wallet).
+
+## Get a Seed From a `.stronghold` file
+
+You can use your `.stronghold` backup file to obtain your wallet's seed.
+
+If don't know where your `.stronghold` file is, click on the `...` menu and select the `Copy system profile location`. 
+
+   ![Copy Stronghold Location](/img/about-iota/iota-wallet/how-to/restore-account/copy-stronghold-location.png)
+
+Once you have the `.stronghold` file at hand, you can drag and drop it into Firefly's Stronghold Backup.
```

Comment:
@begonaalvarezd, I think we may be missing a step here, how does the user reach this menu?

Reply:
If the user is grabbing the stronghold from within Firefly because he/she doesnt have any other backup, the user has to navigate to the location Firefly provided and then drag and drop the `.stronghold` found in the location